JobConfig/pileup/MuStopPileup.fcl

```diff
@@ -54,6 +54,19 @@ physics : {
             spectrumShape: "flat"
           }
           tool_type: "MuCapPhotonGenerator"
+        },
```

Comment:
This configuration is for 'external', does that mean it generates photons? Is that redundant with the flat spectrum above?
What about internal conversion? Should there be a 2nd tool instance for that?

Reply:
Yes, we would need to have a second tool in the list for internal conversions. I can add this as well if this makes sense to you. Its rate will be reduced by 0.0069 with respect to the real photons for reference. I can also consider updating the RMC generator tool to randomly generate internal instead of external conversions at this relative rate if this makes more sense.